public/manifest.json

```diff
@@ -3,7 +3,7 @@
   "manifest_version": 3,
   "name": "Panda Wallet",
   "description": "A non-custodial and open-source wallet for BSV and 1Sat Ordinals",
-  "permissions": ["storage", "activeTab", "tabs", "windows"],
+  "permissions": ["storage", "activeTab", "tabs"],
```

Comment:
I assume removing `windows` will break the "provider" functionality (I believe it relies on it for the popup windows).